ecommerce/enterprise_coupons/utils.py

```diff
@@ -0,0 +1,38 @@
+"""
+Helper methods for enterprise app.
+"""
+import logging
+
+from django.conf import settings
+from .backends import get_coupon_backend
+
+log = logging.getLogger(__name__)
+
+def get_enterprise_catalog_id_from_coupon(coupon_code, backend_alias='default'):
+    """Get catalog ID using configured backend."""
+    backend = get_coupon_backend(backend_alias)
```

Comment:
backend is only used once, so I would recommend to remove it, but for organization purposes, we might leave it. @daviidco